boilerplate/openshift/golang-osd-operator/csv-generate/csv-generate.sh

```diff
@@ -112,7 +112,18 @@ else
 fi
 
 if [[ "$generate_script" = "common" ]] ; then
-    ./boilerplate/openshift/golang-osd-operator/csv-generate/common-generate-operator-bundle.py -o ${operator_name} -d ${OUTPUT_DIR} -p ${OPERATOR_PREV_VERSION} -i ${REPO_DIGEST} -V ${operator_version}
+    # Jenkins can't be relied upon to have py3, so run the generator in
+    # a container.
+    # ...Unless we're already in a container, which is how boilerplate
+    # CI runs. We have py3 there, so run natively in that case.
+    if [[ -z "$CONTAINER_ENGINE" ]]; then
```

Comment:
Will container engine be blank or unset? If so you might want `[[ -z "${CONTAINER_ENGINE+x}" ]]`

Reply:
Blank. But even if it wasn't, with the double quotes it'll always work. We're using this pattern all over the code. It's fine. Promise.

Reply:
Ok boss.

Reply:
Sorry, let me restate: It doesn't matter because we're not running under `-u` (aka `-o nounset`)